WEB/Src/PerformanceCollector/Perf.Shared/AssemblyInfo.cs

```diff
@@ -5,11 +5,7 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-[assembly: InternalsVisibleTo("TestApp40, PublicKey=" + AssemblyInfo.PublicKey)]
-[assembly: InternalsVisibleTo("TestApp45, PublicKey=" + AssemblyInfo.PublicKey)]
```

Comment:
I'm not sure what these TestApp40 and TestApp45 are.
Need to do a quick check and confirm they no longer exist.

Reply:
For TestApp40, I couldn't find any reference.
For TestApp45, we have that in the FunctionalTests solution. It looks its used. i will reverd the 45.

Let me know what do u think about 40.

Reply:
40 looks safe to delete. the 45 we still need to keep. :)

Reply:
ok. i will revert the 45. thanks for reviewing!